pkg/prowgen/jobbase.go

```diff
@@ -160,6 +160,9 @@ func NewProwJobBaseBuilderForTest(configSpec *cioperatorapi.ReleaseBuildConfigur
 		if configSpec.Releases != nil {
 			p.PodSpec.Add(CIPullSecret())
 		}
+		if info.Config.EnableSecretsStoreCSIDriver {
+			p.PodSpec.Add(Arg("enable_secrets_store_csi_driver", "true"))
```

Comment:
`enable_secrets_store_csi_driver` it's ok as a JSON field, but as an argument name I'd rename it to `enable-secrets-store-csi-driver`

Reply:
Yeah I just realized this while writing tests 😄 Fixed now